roles/sap_swpm/tasks/pre_install.yml

```diff
@@ -18,7 +18,7 @@
 - name: SAP SWPM Pre Install - Set sapinst command
   vars:
     sap_swpm_swpm_command_guiserver: "{{ 'SAPINST_START_GUISERVER=false' if not sap_swpm_swpm_observer_mode else '' }}"
-    sap_swpm_swpm_command_observer: "{{ 'SAPINST_REMOTE_ACCESS_USER=' + sap_swpm_swpm_remote_access_user + ' SAPINST_REMOTE_ACCESS_USER_IS_TRUSTED=true' if sap_swpm_swpm_observer_mode and sap_swpm_swpm_remote_access_user is defined and sap_swpm_swpm_remote_access_user | length > 0 else '' }}"
+    sap_swpm_swpm_command_observer: "{{ 'SAPINST_REMOTE_ACCESS_USER=' + sap_swpm_swpm_remote_access_user + ' SAPINST_REMOTE_ACCESS_USER_IS_TRUSTED=true' if sap_swpm_swpm_observer_mode and sap_swpm_swpm_remote_access_user | d('',true) | length > 0  else '' }}"
```

Comment:
I think ansible-lint wants a space after the comma in the default() section. If this is not the case, I'd personally prefer having one, for better readability and also because it appears to be the standard in our repo.

Reply:
done